label_studio/data_export/models.py

```diff
@@ -246,3 +247,12 @@ class Status(models.TextChoices):
         null=True,
         verbose_name=_('created by'),
     )
+
+
+@receiver(post_save, sender=ConvertedFormat)
+def delete_storage_file(sender, instance, **kwargs):
```

Comment:
please don't use signals, write it explicitly in the ConvertedFormat model

Reply:
Fixed